composer.json

```diff
@@ -1,29 +1,31 @@
 {
-    "name": "keboola/my-component",
-    "description": "MyComponent description",
+    "name": "keboola/ex-azure-cost-management",
+    "description": "Exports data from the Azure Cost Management APIs. ",
     "license": "MIT",
     "require": {
         "php": "^7.4",
-        "keboola/php-component": "^8.1"
+        "guzzlehttp/guzzle": "^7.2",
+        "keboola/php-component": "^8.1",
+        "league/oauth2-client": "^2.6"
     },
     "require-dev": {
         "php-parallel-lint/php-parallel-lint": "^1.2",
         "keboola/coding-standard": ">=7.0.2",
-        "keboola/datadir-tests": "^4.0",
+        "keboola/datadir-tests": "^5.2",
         "keboola/php-temp": "^2.0",
         "phpstan/phpstan": "^0.12.14",
         "phpunit/phpunit": "^9.0",
         "symfony/process": "^5.0"
     },
     "autoload": {
         "psr-4": {
-            "MyComponent\\": "src/"
+            "AzureCostExtractor\\": "src/"
```

Comment:
doplň tam pls Keboola -> `Keboola\\AzureCostExtractor\\`

Reply:
:+1: vdaka, to mi nedoslo

Reply:
https://github.com/keboola/ex-azure-cost-management/pull/1/commits/08b720a76c82e6deae5ccd8e1b5e7de99f82706f

Reply:
super